src/main/java/com/ldtteam/structures/blueprints/v1/BlueprintUtil.java

```diff
@@ -228,6 +216,11 @@ public static CompoundNBT writeBlueprintToNBT(final Blueprint schem)
 
         tag.put("mcversion", IntNBT.valueOf(SharedConstants.getVersion().getWorldVersion()));
 
+        tag.put("optional_data", new CompoundNBT());
```

Comment:
make optional_data a constant

Reply:
none of the other tags are constants XD

Reply:
ye but you used this one like 5 times xD

Reply:
from my side no need :D